### ISSUE TITLE
calling metrics-mongodb.rb on a know working mongo server standalone returns no metrics

### ISSUE THREAD

**Issue by nlawson** [sensu-plugins/sensu-plugins-mongodb#19]
calling metrics-mongodb.rb --host localhost  returns no results, tested on a known working standalone mongo server

**Comment by PuKoren** [comment]
+1

MongoDB version: 3.2.1
Storage Engine: WiredTiger

**Comment by eheydrick** [comment]
v1.0.0 has been released with mongo 3.2 support. Can you give that a try and report back.

**Comment by neallawson** [comment]
so, after getting a newer ruby installed, since we were using the embedded ruby from the sensu client, we were able to run the plugin and get some metrics!

```bash
$ metrics-mongodb.rb --host localhost --password xxx --user xxx
ip-10-238-31-106.mongodb.lock.queue.total 0 1465409144
ip-10-238-31-106.mongodb.lock.queue.readers 0 1465409144
ip-10-238-31-106.mongodb.lock.queue.writers 0 1465409144
ip-10-238-31-106.mongodb.connections.current 583 1465409144
ip-10-238-31-106.mongodb.connections.available 50617 1465409144
ip-10-238-31-106.mongodb.cursors.open 2 1465409144
ip-10-238-31-106.mongodb.cursors.timedOut 1 1465409144
ip-10-238-31-106.mongodb.mem.residentMb 3678 1465409144
ip-10-238-31-106.mongodb.mem.virtualMb 6633 1465409144
ip-10-238-31-106.mongodb.mem.mapped 0 1465409144
ip-10-238-31-106.mongodb.mem.pageFaults 1470 1465409144
ip-10-238-31-106.mongodb.asserts.warnings 0 1465409144
ip-10-238-31-106.mongodb.asserts.errors 0 1465409144
```

Thanks!

**Comment by eheydrick** [comment]
Thanks for the followup @neallawson and glad to hear it's working for you.
